src/python/pants/bin/local_pants_runner.py

```diff
@@ -140,6 +141,10 @@ def create(
         if global_bootstrap_options.verify_config:
             options.verify_configs(options_bootstrapper.config)
 
+        # Set PANTS_BIN_NAME, for convenience in user-visible strings. We use `set` in case the user
+        # has set this option in the env as well as at a higher rank
```

Comment:
Is the `set` thing outdated?

Reply:
yeah, posting fixes after self-review soon